crates/matrix-sdk/src/room/mod.rs

```diff
@@ -937,6 +937,20 @@ impl Room {
         self.client.send(request, None).await
     }
 
+    /// Update the tags from the room by calling set_tag or remove_tag method if
+    /// needed.
+    pub async fn update_notable_tags(&self, notable_tags: RoomNotableTags) -> Result<()> {
```

Comment:
I'm not sure this is best to do like this for the future, or offer method for each tags like
`set_is_favorite(bool)`
`set_is_low_priority(bool)`

Reply:
Me neither. @bnjbvr do you have an opinion?

Reply:
I don't find it likely to change multiple ones at the same time, this shouldn't happen quite often so performance isn't critical, and having a single method for each notable tag makes for a better API IMO, so I'd in favor of having `set_is_...` methods :+1:

Reply:
Do you think it's worth checking the local value tags before sending the request? 
I guess we can have some race condition (especially with [this issue](https://github.com/matrix-org/sliding-sync/issues/189)) which could prevent the request to happen when it should.